src/lib/utils/boost_default_memory_resource.cpp

```diff
@@ -3,31 +3,33 @@
 
 #include <cstddef>
 #include <cstdlib>
+#include <iostream>
 
 namespace boost {
 namespace container {
 namespace pmr {
 
 class default_resource_impl : public memory_resource {
  public:
-  virtual ~default_resource_impl() {}
-
   virtual void* do_allocate(std::size_t bytes, std::size_t alignment) { return std::malloc(bytes); }
 
   virtual void do_deallocate(void* p, std::size_t bytes, std::size_t alignment) { std::free(p); }
 
   virtual bool do_is_equal(const memory_resource& other) const BOOST_NOEXCEPT { return &other == this; }
-} default_resource_instance;
+};
```

Comment:
What does this file have to do with the PR?

Reply:
nthng